src/pages/reference/cli/stacks.mdx

```diff
@@ -72,7 +147,7 @@ config:
     memory: 4096
 ```
 
-The provider has been updated from `aws`, `gcp`, and `azure` to `nitric/aws@0.24.0`, `nitric/gcp@0.24.0`, and `nitric/azure@0.24.0` respectively. This change enables locking a version for your deployments, as it was previously tied to the CLI version. This change will also enable creation of your own [custom deployment providers](/reference/providers/custom/building-custom-provider).
+The provider has been updated from `aws`, `gcp`, and `azure` to `nitric/aws@x.x.x`, `nitric/gcp@x.x.x`, and `nitric/azure@x.x.x` respectively. This change enables locking a version for your deployments, as it was previously tied to the CLI version. This change will also enable creation of your own [custom deployment providers](/reference/providers/custom/building-custom-provider).
```

Comment:
This syntax might be a better way to support versioning for `@latest` going forward.

e.g. 0.35.x to get the latest patch revision of version 0.35.